### PR TITLE
Bump rev 1.22.3-asm.1; update canonical service image

### PR DESCRIPTION
**Pull request by sergii-ssh**

### CODE REVIEW

File: asm/canonical-service/controller.yaml [hunk 1]
```diff
@@ -284,6 +284,7 @@ spec:
   selector:
     control-plane: controller-manager
 ---
+---
```
Comment (wonjekang) [inline asm/canonical-service/controller.yaml:287]: Why do we need this?

File: asm/canonical-service/controller.yaml [hunk 2]
```diff
@@ -327,7 +328,7 @@ spec:
         - --enable-leader-election
         command:
         - /manager
-        image: gke.gcr.io/asm/canonical-service-controller:1.10.3-asm.16 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.canonicalServiceHub"}
+        image: gke.gcr.io/asm/canonical-service-controller:1.22.3-asm.1 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.canonicalServiceHub"}
```
Comment (wonjekang) [inline asm/canonical-service/controller.yaml:331]: We don't have this version of image in the image registry. Can you make sure the image exists first?

https://pantheon.corp.google.com/gcr/images/gke-release/global/asm/canonical-service-controller?e=-13802955&inv=1&invt=AbXdYQ&jsmode=o&mods=prod&project=gke-release

File: asm/canonical-service/controller.yaml (hunk 1 above)
Comment (sergii-ssh) [reply asm/canonical-service/controller.yaml]: fixed

File: asm/canonical-service/controller.yaml (hunk 2 above)
Comment (sergii-ssh) [reply asm/canonical-service/controller.yaml:330]: Image is available now

**Review by wonjekang — APPROVED** [review]

**Pull request merged**